README.md

```diff
@@ -618,6 +648,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
 </table>
 
 <!-- ALL-CONTRIBUTORS-LIST:END -->
+<!-- /ix-docs-ignore -->
```

Comment:
Oops, missed this. Will you pull the `## Contributors` header out of the comment here so it appears along with the paragraph below?